frontend/src/components/adminHomeData/adminCalendar.jsx

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import * as React from 'react';
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { CalendarPicker } from '@mui/x-date-pickers/CalendarPicker';
+import Grid from '@mui/material/Grid';
+import '../../css/adminCalendar.css';
+
+export default function AdminCalendar() {
+  const [date, setDate] = React.useState(new Date());
+  return (
+    <div style={{
+      padding: '100px 10px 0px',
+    }}
+    >
+      <LocalizationProvider dateAdapter={AdapterDateFns}>
```

Comment:
Is it possible to connect this calendar with the main calendar? so if they choose a different day, it will change it on the main calendar as well?